### DISCUSSION TITLE
Lesson 11: cannot open generated IPFS image_uri in browser

### DISCUSSION THREAD

**Discussion by Arsenii0** [smartcontractkit/full-blockchain-solidity-course-py#736] [Q&A] [answered]
Hello!
Method `upload_to_ipfs` from Lesson 11 seems working correctly as I could receive the `image_uri` with the generated IPFS hash from the response. 
However, if I follow the generated link, the browser just loops over it and does not open the image (eventually receive 504 Gateway Time-out).

Here is the code (just a little bit modified from the video tutorial. Also I use Kovan instead of Rinkeby):
![image](https://user-images.githubusercontent.com/75928581/148649042-e60c5374-38ef-4fb5-9c8e-e404b5e5f563.png)

Here is the terminal output:
![image](https://user-images.githubusercontent.com/75928581/148649058-2a1f95bd-309c-4686-8e04-485f7135a786.png)

Ipfs daemon is also running successfully

**Comment by cromewar** [comment, marked as answer]
Hello @Arsenii0 I could be a error from the server, As it had passed 9 hours, does the error remains?

**Reply by Arsenii0** [reply]
Yup, still the same

**Reply by cromewar** [reply]
Ok checking again your code I see you are connecting to IPFS through local host, unless you store your ipfs using something like pinata, you would have to be running the ipfs node always on your pc in order to access to it from the browser, could you share your ipfs configuration please.

**Reply by Arsenii0** [reply]
Checked it once again today. And it works, the image is loaded successfully!
You were right, it was an error from the server, thanks for the responses

**Reply by cromewar** [reply]
I'm glad is solved now.
